File: 151003/Mytnik/Lab1/Lab1/Lab1/DTO/NoteRequestTo.cs
using Lab1.DTO.Interface;

namespace Lab1.DTO
{
    public class NoteRequestTo(int Id,int? TweetId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public int? TweetId { get; set; } = TweetId;

        public string? Content { get; set; } = Content;
    }
}

File: 151001/Shcherba/lab1/src/main/java/by/bsuir/dto/NoteRequestTo.java
package by.bsuir.dto;

import jakarta.validation.constraints.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NoteRequestTo {
    private Long id;
    @NotBlank
    @Size(min = 2, max = 4096)
    private String content;
    private Long tweetId;
}
